doc/src/vpr/command_line_usage.rst

```diff
@@ -1517,6 +1517,21 @@ VPR uses a negotiated congestion algorithm (based on Pathfinder) to perform rout
     * `swns` - setup Worst Negative Slack (sWNS) [ns]
     * `stns` - Setup Total Negative Slack (sTNS) [ns]
 
+
+.. option:: --generate_net_timing_report {on | off}
+
+    Generates a net timing report for each net in the design. For each net, the timing information written in the following format:
```

Comment:
Say what the name of the file is (probably base name . something).

Reply:
"net timing report for each net in the design" is a bit vague. Suggest "Generates a report that lists the slack and delay of every routed connection in a design."